packages/twenty-front/src/modules/activities/tasks/components/__stories__/TaskGroups.stories.tsx

```diff
@@ -30,6 +31,7 @@ const meta: Meta<typeof TaskGroups> = {
     ComponentWithRecoilScopeDecorator,
     ObjectMetadataItemsDecorator,
     SnackBarDecorator,
+    I18nFrontDecorator,
```

Comment:
missing i18n provider (provided by this decorator in stories) so useLingui() was failing